### DISCUSSION TITLE
Communication Channel for the LFX project

### DISCUSSION THREAD

**Discussion by literalEval** [finos/a11y-theme-builder#832] [Q&A] [answered]
Hey mentors !!

I am interested in the LFX project "FINOS Accessibility Theme Builder Mentorship". Unfortunately I couldn't find any communication channel to discuss the project with the mentors.
I wanted to discuss the scope of the project before submitting a thorough application. 

Thanks.

**Comment by kris-reed** [comment, marked as answer]
@literalEval This might be useful: https://github.com/finos/a11y-theme-builder/wiki/Communication

**Reply by literalEval** [reply]
Thanks. Forgot about the wiki page somehow. Pardon the silly mistake.

**Reply by aaronreed708** [reply]
Questions whose answers might benefit others might best be put here in GitHub discussions.  But if the questions are more focused on your specific interest, then feel free to reach out to me on Slack.
